### ISSUE TITLE
Separate check-boxes for torrent files and magnet links association

### ISSUE THREAD

**Issue by smaragdus** [aliakseis/LIII#2]
I would appreciate if there are separate check-boxes for torrent files and magnet links association, screen:

![LIII BitTorrent Client 0 1 0 8 - 2019-06-04 - 006](https://user-images.githubusercontent.com/9025757/58840440-5ad5db00-866e-11e9-9c0b-629d5766a03a.png)

I use two torrent clients for torrent files and magnet links- a very old, actually archaic version of **uTorrent** which is very light and thus rather suitable for heavy seeding (currently I am seeding 1500+ torrents) and the latest version of **qBittorrent** for magnet links. So if I decide to replace **qBittorrent** with **LIII BitTorrent Client** I would need to associate only magnet links with it, not torrent files.

If there are separate check-boxes for torrent files and magnet links **LIII BitTorrent Client** would give the user more freedom and flexibility.

**Comment by aliakseis** [comment]
I'm on it.

**Comment by aliakseis** [comment]
Seems to be done. Please note that the default torrent setting affects the Chrome browser after its restarting.

**Comment by smaragdus** [comment]
@aliakseis 

Thank you for implementing my suggestion. I have already downloaded and tested version- [0.1.0.9](https://github.com/aliakseis/LIII/releases/tag/0.1.0.9)- everything seems to be fine. I will close this issue.

**Off-topic**
I discovered **LIII BitTorrent Client** last November and I liked it immediately- C++, open source, clean GUI, ad-free, light-weight. I see in it potential to become a real alternative to the few torrent clients I consider capable nowadays- [qBittorrent](https://www.qbittorrent.org/) (open source but using it in portable mode is a pain) and [Tixati](https://www.tixati.com/) (closed source but it has a native portable version). Since November I plan to suggest some enhancements but due to lack of time I haven't requested them yet (a torrent client is not a note pad for example- it requires far more testing). I hope that this weekend I will have time to have a more thorough look at the current version of **LIII BitTorrent Client**. I appreciate that you continually improve the program. Thank you for your good work and for actively maintaining the project (so many promising projects have been abandoned).
